src/main/java/com/rollbar/android/Notifier.java

```diff
@@ -39,8 +39,8 @@
     private static final String ANDROID = "android";
     private static final String MESSAGE = "message";
 
-
-    private static final int DEFAULT_ITEM_SCHEDULE_DELAY = 1;
+    private static final int DEFAULT_ITEM_SCHEDULE_STARTUP_DELAY = 1;
```

Comment:
I can take a look at it. I'm not totally sure why it was there originally. There may be no use for it. Maybe someone was just trying to avoid a magic number?